Makefile

```diff
@@ -87,6 +88,12 @@ YAML_SRC = \
 LIBIFUPDOWN_${CONFIG_YAML}_OBJ += ${YAML_SRC:.c=.o}
 CPPFLAGS_${CONFIG_YAML} += -DCONFIG_YAML
 
+# enable VRRP crc32 builtin tool (+31 KB). This is stripped version of cksum from coreutils
+CONFIG_CRC32 ?= Y
+CRC32_SRC = tools/crc32.c tools/cksum.c tools/crctab.c
+CRC32_${CONFIG_CRC32}_OBJ += ${CRC32_SRC:.c=.o}
+CRC32_${CONFIG_CRC32} += crc32sum
+
```

Comment:
This configuration code is different from all the others.  It could be simplified, and indeed since there is no way to merge the coreutils version of `cksum` anyway the new version should be made to look like the other applets.

Reply:
I really don't understand how should look as other applets. Do you want to be integrated in ifupdown-ng tool and to looks like an applet like ifup / ifdown / ifquery / ifparse ?

Reply:
Ok. I started to port the tool as applet of ifupdown:

I changed Makefile:
```
# enable VRRP CRC32 builtin tool (+31 KB). This is stripped version of cksum from coreutils
CONFIG_CRC32 ?= Y
CRC32_SRC = cmd/crc32.c cmd/cksum.c cmd/crctab.c
MULTICALL_${CONFIG_CRC32}_OBJ += ${CRC32_SRC:.c=.o}
CMDS_${CONFIG_CRC32} += crc32sum
```